mlrun/kfpops.py

```diff
@@ -821,3 +825,39 @@ def add_function_node_selection_attributes(
             container_op.affinity = function.spec.affinity
 
     return container_op
+
+
+def add_function_security_context(
+    function, container_op: dsl.ContainerOp
+) -> dsl.ContainerOp:
+
+    if (
+        not mlrun.runtimes.RuntimeKinds.is_local_runtime(function.kind)
+        and mlrun.mlconf.function.spec.security_context.enrichment_mode
+        != SecurityContextEnrichmentModes.disabled.value
+    ):
```

Comment:
Instead of having all the function within the `if`, make a negated if and bail early
```suggestion
    if (
        mlrun.runtimes.RuntimeKinds.is_local_runtime(function.kind)
        or mlrun.mlconf.function.spec.security_context.enrichment_mode
        == SecurityContextEnrichmentModes.disabled.value
    ):
        return
```